### ISSUE TITLE
JOSS review: Contribution and authorship

### ISSUE THREAD

**Issue by steko** [tesselle/aion#2]
Hello, I am one of the reviewers for the JOSS submission described at https://github.com/openjournals/joss-reviews/issues/6210 👋

As I go through the checklist, I find this discrepancy:
- in the paper, the author is: Nicolas Frerebeau
- in the documentation (e.g. at https://packages.tesselle.org/aion/authors.html) the authors are listed as: Nicolas Frerebeau and Joe Roe, with Brice Lebrun as "Contributor"

Could you please clarify the authorship, either in the paper or in the documentation? Thanks!

**Comment by nfrerebeau** [comment]
Brice Lebrun is credited as a contributor, as he designed the package logo (I updated the DESCRIPTION file).

Joe Roe is credited as author, as his 'era' package gave me the inspiration for 'aion' (as stated in the paper's acknowledgements). It seems fair to me to credit him in this way. I suggested to Joe Roe that he co-sign the paper, but he declined, considering it not justified (maybe @joeroe can confirm?).
